.circleci/config.yml

```diff
@@ -104,8 +107,14 @@ jobs:
     machine: true
     steps:
       - checkout
-      - run: IMAGE_TAG=${CIRCLE_TAG:-${CIRCLE_BRANCH}} make benchmark
-      - run: IMAGE_TAG=master make benchmark
+      # If the docker creds are not set, the image will not be available, and
+      # we should skip the benchmark step.
+      - run:
+          command: |
+            if ([ -n "${DOCKER_USERNAME}" ] || [ -n "${DOCKER_PASSWORD}" ]); then
```

Comment:
Those values are populated by setting `context: org-global` in the workflow definition. Benchmark never had access to these.